include/psp2kern/kernel/debug.h

```diff
@@ -129,6 +129,69 @@ int ksceKernelGetMinimumAssertionLevel(void);
  */
 int ksceKernelSetMinimumAssertionLevel(int level);
 
+// process
+typedef struct SceKernelDebugEventLog1 { // size is 0x1C
+	int data_0x40;
+	SceUID pid;
+	int budget_type;
+	int data_0x4C;    // 0xA
+	char titleid[0xC];
+} __attribute__((packed)) SceKernelDebugEventLog1;
+
+// Related to network
+typedef struct SceKernelDebugEventLog2 { // size is 0x4
+	int data_0x40;    // 0 or 0x80412118?
+} __attribute__((packed)) SceKernelDebugEventLog2;
+
+// Related to network
+typedef struct SceKernelDebugEventLog3 { // size is 0x54
+	int data_0x40;    // 0?
+	char ip1[0x10];
+	char ip2[0x10];
+	char ip3[0x10];
+	char ip4[0x10];
+	char ip5[0x10];
+} __attribute__((packed)) SceKernelDebugEventLog3;
+
+typedef struct SceKernelDebugEventLog {
+	SceSize size;      // This struct size(variable size)
```

Comment:
```suggestion
	SceSize size;      //!< struct size(variable size)
```
please change inline comment format of struct like this for doxygen.

Reply:
done.